src/html/settings.html

```diff
@@ -221,6 +221,13 @@ <h3>Other settings</h3>
               <span class="description">Select text and start a timer with that description from the right-click menu. If nothing is selected, the page title is used as the description.</span>
             </label>
           </div>
+          <div class="checkbox field">
+            <input type="checkbox" id="dark_mode">
+            <label for="dark_mode">
+              <span class="setting">Enable dark mode</span>
+              <span class="description">Use dark mode for the Toggl Button popups.</span>
```

Comment:
```suggestion
              <span class="description">Use dark mode</span>
```
It affects the settings page as well